### ISSUE TITLE
A11y_ When focus is on 'Steps to Reproduce' edit field screen reader announcing it as 'Please enter details'. It not announcing name of edit field announcing only place holder value.

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#24219]
### Github tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #DSCT1.0.0; #Element:tooltip; #JAWS; #WCAG2.5.3; #ScreenReader; #Win11; #Win32; #NVDA;
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment Details:
Application Name: Database Schema Conversion Toolkit Azure data studio
Version: 1.43.0 (System user)
Windows Version: Win11 Pro 21H2 
OS Build:22000.1817
ScreenReader: JAWS 2023.2303.144 /NVDA 2023.1
### Repro Steps:

1. Login to vm with appropriate credentials
2. Launch Azure data studio.
3. Tab to 'Manage Connection' and press enter
4. Tab to the dash board for Oracle extension
5. Tab to 'Feedback' and press enter.
6. Issue reporter' dialog will open
7. Open AI4D and hover over 'Steps to Reproduce' text box to check its name.

### Actual Result:
When focus is on 'Steps to Reproduce' edit field screen reader announcing it as 'Please enter details'. It not announcing name of edit field announcing only place holder value.
**Similar issue observed for 'Title' edit box in same page.**
### Expected Result:
When focus is on 'Steps to Reproduce' edit field screen reader should announce name of edit field as well as place holder value.

### User Impact:
Speech-input user will not be able to navigate to Steps to Reproduce text field as its name is not as its visual.
### Attachment
![2390355_A11y_Database Schema Conversion Toolkit_Feedback_Feedback_DeveloperTool](https://github.com/microsoft/azuredatastudio/assets/143179527/00e2e9f7-a060-4cc1-8c7e-ee69b07b6085)
[2390355_A11y_Database Schema Conversion Toolkit_Feedback_Feedback_ScreenReaderNVDA.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/bad099e9-c593-4d98-8bab-0aec48703ea8)
[2390355_A11y_Database Schema Conversion Toolkit_Feedback_Feedback_ScreenReaderJAWS.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/40760849-b076-47da-aaeb-5c45cc0ce56b)

**Comment by ShaileshChudawkar** [comment]
Primary bug: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2390355

**Comment by kisantia** [comment]
@ShaileshChudawkar can this issue please be downgraded to sev3 or closed since an upstream issue has been created in vscode and there is no ETA for this to be fixed in the upstream repo?

**Comment by kisantia** [comment]
@ShaileshChudawkar this has now been fixed. Please use the[ latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) to verify the fix. 
Steps to Reproduce:
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/84095572-2215-4af9-8e85-8df4f331aa20)
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/5084774b-8f21-4343-bea9-cc9d837a1eee)

**Comment by kisantia** [comment]
this has been fixed. Please try the[ latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) to verify the fix.

**Comment by PadminiGariki** [comment]
@kisantia 
Verified the issue below environment.
Version: 1.48.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/7bdf5c994bcc25aaeb60584dcaa24f83e6fde7fd
Date: 2024-01-29T22:20:11.186Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.23620

Issue is fixed. Hence the bug is closed.
#Regressed:02-01-24; #Closed;
For reference attachment is added.

[24219_JAWS.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/564cd901-26c2-4b7f-a381-d73068c5fc05)

![MicrosoftTeams-image (6)](https://github.com/microsoft/azuredatastudio/assets/142479541/7ba715da-8944-44fc-b6a4-f01b76d3d3b8)

**Comment by kisantia** [comment]
closing as issue was confirmed fixed above.
